### ISSUE TITLE
Provide property for `download_location` in PyUnsplasy.Photo class.

### ISSUE THREAD

**Issue by salvoventura** [salvoventura/pyunsplash#4]
At this time, the `PyUnsplash.photo` class directly exposes `link_download` which maps to a different property, in particular `photo.links.download`. I can add the additional class property.

However, you can still work around this by something like:
```
import pyunsplash
pu = pyunsplash.PyUnsplash(api_key='<your Unsplash API key>')
photos = pu.photos(type='random', per_page=15)
for photo in photos.entries:
    print photo.links.get('download_location', None)
```

_Originally posted by @salvoventura in https://github.com/salvoventura/pyunsplash/issues/3#issuecomment-448797362_

**Comment by salvoventura** [comment]
Fixed and closed as part of new beta release v1.0.0b7. Please update to this release to get the latest functionality.

**Comment by salvoventura** [comment]
Fixed and closed as part of new beta release v1.0.0b7. Please update to this release to get the latest functionality.
